src/templates/Explore/CommunityPoolsTable/index.tsx

```diff
@@ -59,18 +59,16 @@ type IPoolsInfosProps = {
     timestamp: number
     close: any
   }[]
-  weight_goals: {
-    __typename?: 'WeightGoalPoint' | undefined
-    weights: {
-      __typename?: 'WeightGoal' | undefined
-      asset: {
-        __typename?: 'Asset' | undefined
-        token: {
-          __typename?: 'Token' | undefined
-          logo?: string | null | undefined
-        }
-      }
-    }[]
+  underlying_assets: {
+    token: {
+      logo?: string | null | undefined
```

Comment:
I don't know if this is auto-generated, but there's no need for `undefined` when you've added the `?` in the property.

And if it is auto generated, why is it here? I see that it's a GraphQL Query so can't this be imported from the GraphQL generated folder?